### DISCUSSION TITLE
Create extension layout issue

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#13221] [Archive]
We cant copy the existing tabular layout

We get error when we copy existing tabular layout code and try to create it as new extension(Tabular 2) with small change.

We get the error like this. able to create the dummy layout.
![unknown (5)](https://user-images.githubusercontent.com/36895131/167767368-20034337-5f00-4c39-a646-24110099ce51.png)

Point 1 : How we will make small change in the directus tabular layout in Directus cloud/Cloudron deployment 

### To Reproduce

1. Copy the existing tabular layout code from directus existing tabular layout
2. Add it to the the new folder
3. Create the new extension using directus create extension  sdk 

<details>
<summary>More info</summary>

### Errors Shown

Error while compiling it

### What version of Directus are you using?

9.7.1

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres

### What browser are you using?

Chrome

### How are you deploying Directus?

Cloudron

</details>

**Comment by azrikahar** [comment]
Seems like you are running `npm run build` in your forked Directus repo directory rather than the new extension project directory you mentioned in step 3, so it's running something very different (vite build) rather than actually building your extension (which _should_ be rollup instead). Do make sure you are running the command in the correct path, depending on where did you paste the code you copied 👍

**Comment by rijkvanzanten** [comment]
Also, the core layouts might use internal code that extensions don't have access to, so you can't just copy paste everything and expect it to work flawlessly 👍🏻

**Reply by Jeyakumarrathnaselvi** [reply]
But how users of directus cloud will make if we need any small changes in existing layout.
